main.tf

```diff
@@ -7,6 +7,8 @@ locals {
 resource "google_container_cluster" "cluster" {
   count = var.module_enabled ? 1 : 0
 
+  provider = google-beta
+
   project = local.project
```

Comment:
please drop local project loading... the provider is falling back automatically..